src/SonarScanner.MSBuild.PostProcessor/PostProcessor.cs

```diff
@@ -74,7 +74,7 @@ public virtual bool Execute(string[] args, AnalysisConfig config, IBuildSettings
             return false;   // logging already done
         }
 
-        var analysisResult = CreateAnalysisResult(config);
+        var analysisResult = CreateAnalysisResult(config, provider);
```

Comment:
Please rename `provider` to `cmdLineArgs` like in the usage. The `provider` is not clear enough in this class. I didn't like it before, but only now I really hate it and have a good new name for it